### ISSUE TITLE
Confusion about finetune_cls.sh

### ISSUE THREAD

**Issue by TUDelftHao** [2U1/Qwen-VL-Series-Finetune#175]
Thank you for this amazing project!

While I am going to conduct the QwenVL classification task, I have some questions about the parameters used in finetune_cls.sh:
1. What is the global batch size used for the default learning rate?
2. Why is the head_lr different from learning_rate? Is this the empirical set？ 
3. Why are you setting the llm frozen while the vision tower and merger module trainable, which is in contrast to the setup of sft the mllm (usually we only finetune the LLM part).

**Comment by TUDelftHao** [comment]
Thank you so much for the helpful reply!

**Comment by 2U1** [comment]
@TUDelftHao I've had some research and found that mlp could help when doing classification task using models like Qwen.
I've made some options and unfreezing only some of the layers. So you could try various options for it.

For making a baseline I would recommend unfreezing all the layer and use single layer head. Use `cross_entropy` for the loss.

```
    --learning_rate 8e-6 \
    --head_lr 1.6e-5 \
    --vision_lr 1.6e-6 \
    --merger_lr 1.6e-6 \
```

This could be a good start when using `adamw_bnb_8bit` optimizer. When you are using `adamaw_torch_fused` or default adam, then increase the lrs about 20%.
